src/software/new_geom/util/distance_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include "software/new_geom/util/distance.h"
+
+#include <gtest/gtest.h>
+
+#include "software/new_geom/line.h"
+#include "software/new_geom/point.h"
+
+TEST(DistanceTest, vertical_line_point)
+{
+    Line l(Point(0, 1), Point(0, 0));
+    Point p(2, 0);
+    EXPECT_DOUBLE_EQ(distance(l, p), 2);
+    EXPECT_DOUBLE_EQ(distance(p, l), 2);
+}
+
+TEST(DistanceTest, line_point1)
```

Comment:
Same comment as above re: descriptive test names

Reply:
Changed to be more descriptive and added a few more test cases